### PR TITLE
Added an example for`api/report` with fake data

### PR DESCRIPTION
**Pull request by shanewholloway-usds**
Transformed Jim's tech lineage demonstration into example documentation.

## Changes
- Added `prime-router/examples/upload-synthetic-data` with markdown readme and example shell script
- Typo fix for `devenv-instrastructure.sh` to refer to the renamed `docker-infrastructure.yml` file.

## Checklist
- [x] Tested locally?

### CODE REVIEW

File: prime-router/examples/upload-synthetic-data/readme.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+### Examine
+
+View the local download page: [http://localhost:7071/api/download](http://localhost:7071/api/download)
+
+Inspect the local devenv PostgreSQL database using `psql` or
+using [`adminer`](http://localhost:8080/?pgsql=db_pgsql&username=prime&db=prime_data_hub&ns=public).
+Connection details can be found in [docker-infrastructure.yml](../../docker-infrastructure.yml)
+
```
Comment (jimduff-usds) [inline prime-router/examples/upload-synthetic-data/readme.md:80]: I have some useful queries (the ones I demo'd last Friday) in here:
```
docs/proposals/0003-lineage/descendant-queries.sh
docs/proposals/0003-lineage/ancestor-queries.sh
```
But `docs` is a weird place to put queries, so I may move these to your `examples` area.

File: prime-router/examples/upload-synthetic-data/readme.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# build and run PRIME Data Hub
+docker-compose up -d
+
+# optionally, follow logs of `prime_dev` container
+docker-compose logs -f prime_dev
+```
+
+### Generate synthetic data
+
+Use `prime data` command to generate synthetic data.
```
Comment (jimduff-usds) [inline prime-router/examples/upload-synthetic-data/readme.md:25]:
```suggestion
Use `prime data` command to generate fake data.
```

I think we are distinguishing between 'fake' data and Mo's new 'synthetic' data generator, which at this moment is still in his PR branch #404.

This command as written generates fake data, so we should probably label it that way. (I think the intention is that Mo's --synthesize command is you start with real data, and transmogrify it.)

**Review by jimduff-usds — APPROVED** [review]
Thanks, Shane, I really like the idea of developing a library or knowledge base of examples.   I had one minor change suggestion. 

Also, @RickHawesUSDS just recently created docs/playbooks - can you consider if examples/ is close enough to playbooks/ that they should be consolidated together, or not?

File: prime-router/examples/upload-synthetic-data/readme.md (hunk 2 above)
Comment (shanewholloway-usds) [reply prime-router/examples/upload-synthetic-data/readme.md:25]: I didn't understand that distinction and the nuance between them -- thank you! I've updated the folder name and all other references of _synthetic_ to _fake_ as well. 

Another example might be worthwhile for me to explain the difference.

**Comment by shanewholloway-usds** [comment]
Thanks for the review @jimduff-usds!

My distinction between `examples` and `docs/playbooks` is the intended audience. The `examples` folder is targeted at API integrators, inside-the-product development, etc. The `playbooks` collection appears targeted at the operational audience -- which is our team for now -- focused on database operations, uptime, communications, backups, etc. 

Examples are executable documentation with the aim of introducing a specific product / system concept. Useful for external collaborators and onboarding developers to see how things work. Being executable, you can run it quickly to find out if the "documentation" still works, and one can find places to put breakpoints and otherwise explore interactively. Examples also provides a good place for "tech demos" so other developers on the project can have an easy on-ramp to our work.

Both intents are similar in leaving a marked trail to follow.

File: prime-router/examples/upload-synthetic-data/readme.md (hunk 2 above)
Comment (shanewholloway-usds) [reply prime-router/examples/upload-synthetic-data/readme.md]: Fixed in 3da3651d8d19f269cab03a6bac461b958660827d

File: prime-router/examples/upload-synthetic-data/readme.md (hunk 1 above)
Comment (shanewholloway-usds) [reply prime-router/examples/upload-synthetic-data/readme.md]: I think the lineage queries would make a wonderful second example.

Comment (shanewholloway-usds) [reply prime-router/examples/upload-synthetic-data/readme.md]: Started a lineage example in #432

**Pull request merged**